### DISCUSSION TITLE
Welcome to AMDC-Hardware Discussions!

### DISCUSSION THREAD

**Discussion by elsevers** [Severson-Group/AMDC-Hardware#196] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

This discussion board is centered around the AMDC hardware. Other discussion boards related to the AMDC community include:
   * [Docs](https://github.com/Severson-Group/docs.amdc.dev/discussions)
   * [Firmware](https://github.com/Severson-Group/AMDC-Firmware/discussions)

**Comment by Almenau14** [comment]
Olá, sou brasileiro e estudante de engenharia elétrica e estou buscando realizar a confecção da PCB com base na ultima versão disponível a REV20231005F. Mas gostaria de saber se vocês mesmos confeccionaram a placa de circuito impresso ou assim como eu pensaram em mandar fazer na china. Apenas gostaria de saber se é confiável a confecção da PCB pela china. Por se tratar de uma placa com 6 layers e com bastante detalhes.

**Reply by npetersen2** [reply]
Hi, welcome to the AMDC!

If I understand your question, yes, the hardware design is relatively standard and uses standard PCB manufacturing design rules. This enables it to be assembled at most any PCB fab houses. We mostly order the AMDCs as fully assembled. We have had good luck with PCBWay in China.

See: https://docs.amdc.dev/hardware/obtaining-hardware.html

**Reply by Almenau14** [reply]
thank you!
I send the gerbers to PCBway, and now I am waiting review.

**Comment by Almenau14** [comment]
Hi, the PCBWay send me this message!

"The correct package for the listed Part Number [TPS22965DSGT] is indeed [WSON-8-EP(2x2)], not [SON50P200X80_HS-9N].Please confirm."
About the "U30" component of AMDC Hardware, it's corretcly them afirmation?

**Reply by npetersen2** [reply]
Hello, can you please open a new discussion thread for this concern? Thanks.
